packages/flutter/lib/src/material/filled_button.dart

```diff
@@ -524,10 +532,11 @@ class _FilledButtonWithIcon extends FilledButton {
 }
 
 class _FilledButtonWithIconChild extends StatelessWidget {
-  const _FilledButtonWithIconChild({ required this.label, required this.icon });
+  const _FilledButtonWithIconChild({ required this.label, required this.icon, required this.iconAlignment });
```

Comment:
```suggestion
  const _FilledButtonWithIconChild({
    required this.label,
    required this.icon,
    required this.iconAlignment,
  });
```

This is exceeding the `80` char limit.